### PR TITLE
Deprecate duplicated functions

### PR DESCRIPTION
**Pull request by wlsf82**
Deprecate functions that were replaced by other functions on version 3.7.0.

> **Do not merge after approval due to the publishing process.**

## Pull Request Checklist
	
	- [x] Have you followed the steps in our [Contributing](https://github.com/wlsf82/protractor-helper/blob/master/docs/CONTRIBUTING.md) document?
	- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/wlsf82/protractor-helper/pulls) for the same update/change?
	- [x] Have you successfully ran tests with your changes locally?
	- [-] Have you written new tests for your changes, if necessary?
	- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
	- [x] Have you made improvements to the documentation, if necessary? ([docs/](https://github.com/wlsf82/protractor-helper/tree/master/docs) folder and [README.md](https://github.com/wlsf82/protractor-helper/blob/master/README.md))
	- [-] Have you linked the correct [GitHub issue](https://github.com/wlsf82/protractor-helper/issues), if there is any?
	
	> Pull requests that do not pass the automated tests on [SemaphoreCI](http://semaphoreci.com) and the code quality verification on [Better Code Hub](https://bettercodehub.com/) will not be reviewed

### CODE REVIEW

File: test/protractor.conf.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 const constants = require("./constants");
 
 module.exports.config = {
+  directConnect: true,
```
Comment (PauloGoncalvesBH) [inline test/protractor.conf.js]: Why did you include this line?

**Comment by PauloGoncalvesBH** [comment]
I clicked on a wrong button and deleted my review... I will made again.

File: README.md [hunk 2]
```diff
@@ -25,27 +25,27 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
   - [`waitForElementNotToBePresent`](#waitforelementnottobepresent)
   - [`waitForElementVisibility`](#waitforelementvisibility)
   - [`waitForElementNotToBeVisible`](#waitforelementnottobevisible)
-  - [`clickWhenClickable`](#clickwhenclickable)
+  - [`clickWhenClickable` (deprecated)](#clickwhenclickable)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 3]
```diff
@@ -25,27 +25,27 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
...
   - [`click`](#click)
-  - [`fillFieldWithTextWhenVisible`](#fillfieldwithtextwhenvisible)
+  - [`fillFieldWithTextWhenVisible` (deprecated)](#fillfieldwithtextwhenvisible)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 4]
```diff
@@ -25,27 +25,27 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
...
   - [`fillFieldWithText`](#fillfieldwithtext)
-  - [`fillInputFieldWithFileWhenPresent`](#fillinputfieldwithfilewhenpresent)
+  - [`fillInputFieldWithFileWhenPresent` (deprecated)](#fillinputfieldwithfilewhenpresent)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 5]
```diff
@@ -25,27 +25,27 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
...
   - [`uploadFileIntoInputField`](#uploadfileintoinputfield)
-  - [`clearFieldWhenVisible`](#clearfieldwhenvisible)
+  - [`clearFieldWhenVisible` (deprecated)](#clearfieldwhenvisible)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 6]
```diff
@@ -25,27 +25,27 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
...
   - [`clear`](#clear)
-  - [`clearFieldWhenVisibleAndFillItWithText`](#clearfieldwhenvisibleandfillitwithtext)
+  - [`clearFieldWhenVisibleAndFillItWithText` (deprecated)](#clearfieldwhenvisibleandfillitwithtext)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 7]
```diff
@@ -25,27 +25,27 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
...
   - [`clearFieldAndFillItWithText`](#clearfieldandfillitwithtext)
-  - [`tapWhenTappable`](#tapwhentappable)
+  - [`tapWhenTappable` (deprecated)](#tapwhentappable)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 8]
```diff
@@ -25,27 +25,27 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
...
   - [`tap`](#tap)
   - [`waitForTextToBePresentInElement`](#waitfortexttobepresentinelement)
   - [`waitForTextNotToBePresentInElement`](#waitfortextnottobepresentinelement)
   - [`waitForUrlToBeEqualToExpectedUrl`](#waitforurltobeequaltoexpectedurl)
   - [`waitForUrlNotToBeEqualToExpectedUrl`](#waitforurlnottobeequaltoexpectedurl)
   - [`waitForUrlToContainString`](#waitforurltocontainstring)
   - [`waitForUrlNotToContainString`](#waitforurlnottocontainstring)
-  - [`fillFieldWithTextWhenVisibleAndPressEnter`](#fillfieldwithtextwhenvisibleandpressenter)
+  - [`fillFieldWithTextWhenVisibleAndPressEnter` (deprecated)](#fillfieldwithtextwhenvisibleandpressenter)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 9]
```diff
@@ -25,27 +25,27 @@ Many of the helper functions on this library uses `protractor.ExpectedConditions
...
   - [`fillFieldWithTextAndPressEnter`](#fillfieldwithtextandpressenter)
-  - [`scrollToElementWhenVisible`](#scrolltoelementwhenvisible)
+  - [`scrollToElementWhenVisible` (deprecated)](#scrolltoelementwhenvisible)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 10]
```diff
@@ -200,7 +200,7 @@ This method waits for an element to be visible in the page. Being displayed mean
 This method is the opposite of the previous one, so, it waits for an element not to be visible in the page. By saying not being displayed means that the element may be in the DOM, but not visible.
 [Example](docs/EXAMPLES.md#waitforelementnottobevisible)
 
-### `clickWhenClickable`
+### `clickWhenClickable` (deprecated)
 
 This method is used to click in an element as soon as it is in a clickable state. This means that the element is visible and enabled for clicking.
 [Example](docs/EXAMPLES.md#clickwhenclickable)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 11]
```diff
@@ -212,7 +212,7 @@ This method is used to click in an element as soon as it is in a clickable state
 This method is used to click in an element as soon as it is in a clickable state. This means that the element is visible and enabled for clicking.
 [Example](docs/EXAMPLES.md#click)
 
-### `fillFieldWithTextWhenVisible`
+### `fillFieldWithTextWhenVisible` (deprecated)
 
 This method fills an input field with a text as soon as such field is visible.
 [Example](docs/EXAMPLES.md#fillfieldwithtextwhenvisible)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 12]
```diff
@@ -224,7 +224,7 @@ This method fills an input field with a text as soon as such field is visible.
 This method fills an input field with a text as soon as such field is visible.
 [Example](docs/EXAMPLES.md#fillFieldWithText)
 
-### `fillInputFieldWithFileWhenPresent`
+### `fillInputFieldWithFileWhenPresent` (deprecated)
 
 This method fills a file input field with a specified file as soon as the file input field is present in the DOM.
 [Example](docs/EXAMPLES.md#fillinputfieldwithfilewhenpresent)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 13]
```diff
@@ -236,7 +236,7 @@ This method fills a file input field with a specified file as soon as the file i
 This method uploads a file in a file input field as soon as the file input field is present in the DOM.
 [Example](docs/EXAMPLES.md#uploadFileIntoInputField)
 
-### `clearFieldWhenVisible`
+### `clearFieldWhenVisible` (deprecated)
 
 This method clears a text input field as soon as such field is visible.
 [Example](docs/EXAMPLES.md#clearFieldWhenVisible)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 14]
```diff
@@ -248,7 +248,7 @@ This method clears a text input field as soon as such field is visible.
 This method clears a text input field as soon as such field is visible.
 [Example](docs/EXAMPLES.md#clear)
 
-### `clearFieldWhenVisibleAndFillItWithText`
+### `clearFieldWhenVisibleAndFillItWithText` (deprecated)
 
 This method clears a text input field as soon as such field is visible, and then it fills it with a text.
 [Example](docs/EXAMPLES.md#clearfieldwhenvisibleandfillitwithtext)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 15]
```diff
@@ -260,7 +260,7 @@ This method clears a text input field as soon as such field is visible, and then
 This method clears a text input field as soon as such field is visible, and then it fills it with a text.
 [Example](docs/EXAMPLES.md#clearFieldAndFillItWithText)
 
-### `tapWhenTappable`
+### `tapWhenTappable` (deprecated)
 
 This method performs a tap action on a clickable/tappable HTML element as soon as it is in a clickable/tappable state. This method is used when performing web mobile testing in mobile emulators, for example.
 [Example](docs/EXAMPLES.md#tapwhentappable)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 16]
```diff
@@ -302,7 +302,7 @@ This method waits for the URL to contain an expected string. Such method is usef
 This method waits for the URL not to contain an expected string. Such method is useful when you want to perform verifications based on the current URL.
 [Example](docs/EXAMPLES.md#waitforurlnottocontainstring)
 
-### `fillFieldWithTextWhenVisibleAndPressEnter`
+### `fillFieldWithTextWhenVisibleAndPressEnter` (deprecated)
 
 This method fills an input field with a text as soon as such field is visible and then it simulates pressing the ENTER key from the keyboard. This method is useful in cases such as when doing a search and pressing the ENTER key, instead of having to fill the input field and clicking the search button, for example.
 [Example](docs/EXAMPLES.md#fillfieldwithtextwhenvisibleandpressenter)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

File: README.md [hunk 17]
```diff
@@ -314,7 +314,7 @@ This method fills an input field with a text as soon as such field is visible an
 This method fills an input field with a text as soon as such field is visible and then it simulates pressing the ENTER key from the keyboard. This method is useful in cases such as when doing a search and pressing the ENTER key, instead of having to fill the input field and clicking the search button, for example.
 [Example](docs/EXAMPLES.md#fillFieldWithTextAndPressEnter)
 
-### `scrollToElementWhenVisible`
+### `scrollToElementWhenVisible` (deprecated)
 
 This method is used to scroll up to an element on the page as soon as the element is visible in the DOM.
 [Example](docs/EXAMPLES.md#scrolltoelementwhenvisible)
```
Comment (PauloGoncalvesBH) [inline README.md]: That link don't work.

**Comment by PauloGoncalvesBH** [comment]
I'm working on a new solution to that. I'll create a new PR that week.

**Comment by wlsf82** [comment]
Closes in favor of https://github.com/wlsf82/protractor-helper/issues/24.

**Pull request closed without merging**